# Inputs
string table_name
string entry_name
string value
---
# Outputs
bool success